mache/spack/compy_intel_impi.yaml

```diff
@@ -129,22 +129,22 @@ spack:
       buildable: false
     netcdf-c:
       externals:
-      - spec: netcdf-c~mpi~parallel-netcdf@4.6.3
-        prefix: /share/apps/netcdf/4.6.3/intel/19.0.5
+      - spec: netcdf-c+mpi~parallel-netcdf@4.6.3
+        prefix: /share/apps/netcdf/4.6.3/intel/20.0.0
```

Comment:
@altheaden, can you use a hard-reset to remove the last commit I made?  It turns out not to be correct.  NetCDF is still being built without MPI support.

Reply:
Sure thing!

Reply:
Thanks!